.github/component_owners.yml

```diff
@@ -1,19 +1,26 @@
 components:
+  detectors/node/opentelemetry-resource-detector-aws:
+    - NathanielRN
+    - willarmiros
+  detectors/node/opentelemetry-resource-detector-alibaba-cloud:
+    - legendecas
   packages/opentelemetry-id-generator-aws-xray:
     - NathanielRN
     - willarmiros
   plugins/node/opentelemetry-instrumentation-aws-lambda:
     - NathanielRN
     - willarmiros
-  plugins/node/opentelemetry-instrumentation-restify:
+  plugins/node/opentelemetry-instrumentation-generic-pool:
     - rauno56
-  plugins/node/opentelemetry-instrumentation-router:
+  plugins/node/opentelemetry-instrumentation-knex:
     - rauno56
   plugins/node/opentelemetry-instrumentation-memcached:
     - rauno56
-  plugins/node/opentelemetry-instrumentation-knex:
+  plugins/node/opentelemetry-instrumentation-nestjs-core:
```

Comment:
Can you update it to be `Rauno56`? At the current time the action is case sensitive